docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 
+* :feature:`6688` Add support for Woo Centralized Exchange
```

Comment:
It's fine to add since you already did (and exchange work for frontend is small) but we normally let frontend (and remind them to) write the changelog once they finish their work, otherwise it may happen we add somehing to changelog that's not ready yet. Keep it in mind please.